.travis.yml

```diff
@@ -1,4 +1,14 @@
 language: minimal
 
+dist: bionic
+
+services: docker
+
+install:
+  - make build
+
 script:
-  - 'true'
+  - make start
```

Comment:
you can specify multiple targets in one line with `make start wait-healthy`, if it makes it easier